etc/skel/.config/hypr/config/keybinds.conf

```diff
@@ -40,53 +40,53 @@ bindd = , XF86AudioPrev, Previous track, exec, playerctl previous
 ## Resizing windows
 # Activate keyboard window resize mode
-# https://wiki.hyprland.org/0.45.0/Configuring/Binds/#submaps
+# https://wiki.hyprland.org/Configuring/Binds/#submaps
 bindd = $mainMod, R, Activates window resizing mode, submap, resize
 submap = resize
-bindde = , right, Resize to the right (resizing mode), resizeactive, 15 0
-bindde = , left, Resize to the left (resizing mode), resizeactive, -15 0
-bindde = , up, Resize upwards (resizing mode), resizeactive, 0 -15
-bindde = , down, Resize downwards (resizing mode), resizeactive, 0 15
-bindde = , l, Resize to the right (resizing mode), resizeactive, 15 0
-bindde = , h, Resize to the left (resizing mode), resizeactive, -15 0
-bindde = , k, Resize upwards (resizing mode), resizeactive, 0 -15
-bindde = , j, Resize downwards (resizing mode), resizeactive, 0 15
+bindd = , right, Resize to the right (resizing mode), resizeactive, 15 0
+bindd = , left, Resize to the left (resizing mode), resizeactive, -15 0
+bindd = , up, Resize upwards (resizing mode), resizeactive, 0 -15
+bindd = , down, Resize downwards (resizing mode), resizeactive, 0 15
+bindd = , l, Resize to the right (resizing mode), resizeactive, 15 0
+bindd = , h, Resize to the left (resizing mode), resizeactive, -15 0
+bindd = , k, Resize upwards (resizing mode), resizeactive, 0 -15
+bindd = , j, Resize downwards (resizing mode), resizeactive, 0 15
 bindd = , escape, Ends window resizing mode, submap, reset
 submap = reset
 # Quick resize window with keyboard
 # !!! added $mainMod here because CTRL + SHIFT is used for word selection in various text editors
-bindde = $mainMod CTRL SHIFT, right, Resize to the right, resizeactive, 15 0
-bindde = $mainMod CTRL SHIFT, left, Resize to the left, resizeactive, -15 0
-bindde = $mainMod CTRL SHIFT, up, Resize upwards, resizeactive, 0 -15
-bindde = $mainMod CTRL SHIFT, down, Resize downwards, resizeactive, 0 15
-bindde = $mainMod CTRL SHIFT, l, Resize to the right, resizeactive, 15 0
-bindde = $mainMod CTRL SHIFT, h, Resize to the left, resizeactive, -15 0
-bindde = $mainMod CTRL SHIFT, k, Resize upwards, resizeactive, 0 -15
-bindde = $mainMod CTRL SHIFT, j, Resize downwards, resizeactive, 0 15
+bindd = $mainMod CTRL SHIFT, right, Resize to the right, resizeactive, 15 0
+bindd = $mainMod CTRL SHIFT, left, Resize to the left, resizeactive, -15 0
+bindd = $mainMod CTRL SHIFT, up, Resize upwards, resizeactive, 0 -15
+bindd = $mainMod CTRL SHIFT, down, Resize downwards, resizeactive, 0 15
+bindd = $mainMod CTRL SHIFT, l, Resize to the right, resizeactive, 15 0
+bindd = $mainMod CTRL SHIFT, h, Resize to the left, resizeactive, -15 0
+bindd = $mainMod CTRL SHIFT, k, Resize upwards, resizeactive, 0 -15
+bindd = $mainMod CTRL SHIFT, j, Resize downwards, resizeactive, 0 15
 # Resize window with mainMod + LMB/RMB and dragging
-binddm = $mainMod, mouse:273, Resize the window towards a direction, resizewindow
+bindd = $mainMod, mouse:273, Resize the window towards a direction, resizewindow
```

Comment:
`resizewindow` no longer exists, i think it got replaced by `resizeactive`

Reply:
It does exist, my dumb brain just decided to fully ignore the fact that we're using the mouse to resize/move the window.